agent/src/main/java/com/teamscale/jacoco/agent/PreMain.java

```diff
@@ -55,15 +58,16 @@ public static void premain(String options, Instrumentation instrumentation) thro
 		System.setProperty(LOCKING_SYSTEM_PROPERTY, "true");
 
 		String environmentConfigId = System.getenv(CONFIG_ID_ENVIRONMENT_VARIABLE);
-		if (StringUtils.isEmpty(options) && environmentConfigId == null) {
+		String environmentConfigFile = System.getenv(CONFIG_FILE_ENVIRONMENT_VARIABLE);
+		if (StringUtils.isEmpty(options) && environmentConfigId == null && environmentConfigFile == null) {
 			// profiler was registered globally and no config was set explicitly by the user, thus ignore this process
 			// and don't profile anything
 			return;
 		}
 
```

Comment:
I just tested locally what happens if you set both environment variables. On windows, it looks like the config-file variable takes priority, and the config from Teamscale declared with the ID will be ignored. While this is a very artificial problem (as we only need the config file variable if the config id does not work), it would be good to log something in case both are set. 
However, I can imagine this could be confusing, if a customer switches from the config file system to the config id system and we set the id variable and have to wonder why that does not work. WDYT?

Reply:
Perhaps this can be done in the AgentOptionsParser instead.